docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -6,13 +6,55 @@
    "source": [
     "# Quantum Kernel Machine Learning\n",
     "\n",
-    "The general task of machine learning is to find and study patterns in data. For many datasets, the datapoints are better understood in a higher dimensional feature space, through the use of a kernel function:\n",
+    "## Overview\n",
+    "\n",
+    "Machine learning and quantum computing are two technologies each with the potential for altering how computation is performed to address previously unsolvable problems. Kernel methods for machine learning are good for pattern recognition, with support vector machines (SVMs) being the most well-known method for classification problems.\n",
```

Comment:
I am not too convinced by the initial sentence, I think that we could use the old introduction with a bit more context instead:

```suggestion
    "The general task of machine learning is to find and study patterns in data. For many datasets, the datapoints are better understood in a higher dimensional feature space, through the use of a kernel function. This tutorial will show how construct and use quantum kernel functions in the context of support vector machines (SVMs) for a series of common machine learning tasks. \n",
```